contrib/ipp/CMakeLists.txt

```diff
@@ -46,3 +53,6 @@ list( APPEND sources ${srcs})
 enable_language(C ASM)
 set( CMAKE_ASM_SOURCE_FILE_EXTENSIONS s )
 make_library( ipp none static )
+
+SET( ASM_OPTIONS "-x assembler-with-cpp")
+SET( CMAKE_ASM_FLAGS "${CFLAGS} ${ASM_OPTIONS}" )
```

Comment:
hm. Why we did not have this before anywhere in the code?

Also:
1. You don't need separately set ASM_OPTIONS I believe, you can do that within same CMAKE_ASM_FLAGS line
2. You actually override user-specified asm flags, you need to use:
list(APPEND CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} -x assembler-with-cpp")
3. Why you add CFLAGS here?

Reply:
Done. I thought that if we combine C and ASM, then we need some C flags for the ASM, but it seems they are not really needed